testing_and_setup/compass/ocean/coastal/USDEQU120cr10rr2/build_mesh/define_base_mesh.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+
+import numpy as np
+import coastal_tools as ct
+
+
+def cellWidthVsLatLon():
+    km = 1000.0
+
+    params = ct.default_params
```

Comment:
@sbrus89, could you please provide some comments here?  I think you are layering different levels of mesh refinement here but it would be really helpful to have more clarity here.

Reply:
Sure, this definitely needs it.

Reply:
Thanks!